doc: fix openshift project link and wording about catalog (#6)
integr8ly/solution-patterns: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

//Home > Search > Select Templates from the drop down > Search for datasync and they should show up. Just make sure you're on the openshift namespace

. Go to the link:{openshift-host}/topology/ns/{walkthrough-namespace}[project console, window="_blank"] in OpenShift.
. Click the *From Catalog* button.
. Search the developer catalog for 'Data Sync Showcase'.
. Click on the 'Data Sync Showcase' entry to add it to your project.

> . Accept all the defaults of the provisioning wizard and click *Create*.

Reviewer: @darahayes I had to select the "Instantiate Template" button first to kick this off. Did you have to do that as well?

Reviewer 2: This is true. We need to add this step

```suggestion
. Select "Instantiate Template"
. Accept all the defaults of the provisioning wizard and click *Create*.
```

> . Accept all the defaults of the provisioning wizard and click *Create*.

Reviewer: @darahayes This was a form on my screen instead of a wizard.

Reviewer 2: ```suggestion
. Accept all the defaults of the provisioning form and click *Create*.
```

> . Check the *Topology* screen of your project and wait until three pods are running:

Reviewer: @darahayes  How do I get to this screen from the "Template Instance Details" screen?

Reviewer 2: It think we should use the direct link to pods here.

```suggestion
. Go to link:{openshift-host}/k8s/ns/{walkthrough-namespace}/pods screen of your project and wait until three pods are running:
```
